packages/sfui/frameworks/vue/components/SfSelect/SfSelect.vue

```diff
@@ -66,7 +78,10 @@ const changedValue = (event: Event) => {
         },
       ]"
       data-testid="select-input"
+      @blur="onBlurSelect"
+      @click="onClickSelect"
       @change="changedValue"
+      @keydown="onKeydownSelect"
```

Comment:
What about `space` modifier? When added you could remove condition in `onKeydownSelect` handler.
```suggestion
      @keydown.space="onKeydownHandler"
```

Reply:
5b1b058aea610378984cf43effa1ab3a76cdbd54